build.sh

```diff
@@ -4,6 +4,7 @@ mvn -Djava.awt.headless=true clean package
 pushd angularjs-portal-frame
 mvn -Djava.awt.headless=true tomcat7:redeploy
 popd
+mvn install
```

Comment:
since `install` implies `package`, probably cleaner to just make the first `mvn` line

```
mvn -Djava.awt.headless=true clean install
```